File: createJson/src/main/java/com/ruzhan/jsonfile/data/WeGotNext.java
package com.ruzhan.jsonfile.data;

import com.ruzhan.jsonfile.model.Introduce;
import com.ruzhan.jsonfile.model.Movie;
import com.ruzhan.jsonfile.model.MovieDetail;
import com.ruzhan.jsonfile.model.Video;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by ruzhan123 on 2018/7/3.
 */
public class WeGotNext {

    private static final int ID = 1303;
    private static final String PLAY_COUNT = "1";

    private static final String TITLE = "We Got Next 户外篮球纪录片";
    private static final String TAG = "体育·篮球·纪录片";

    private static final String IMAGE = "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/we-got-next.jpg";

    private static final String DESC = "《体育画报》五人组奔赴美国20天，只为探索一个美国街头篮球的真相。";

    public static final Movie movie = new Movie();
    public static final MovieDetail movieDetail = new MovieDetail();

    private static final List<Introduce> introduceList = new ArrayList<>();
    private static final List<Video> videoList = new ArrayList<>();

    static {
        introduceList.add(new Introduce(Introduce.TEXT,
                DESC, ""));

        introduceList.add(new Introduce(Introduce.IMAGE, "",
                "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/we-got-next01.jpg"));

        introduceList.add(new Introduce(Introduce.TEXT,
                "接下来的36分钟，让我们一起踏上美国最传奇的球场，与传奇对话。愿36分钟后，当你重新投出手中的篮球，感觉从此不同。", ""));

        videoList.add(new Video(String.valueOf(ID + 11),
                "We Got Next 户外篮球纪录片", IMAGE, PLAY_COUNT, 1, "电影",
                "http://player.bilibili.com/player.html?aid=26016313&cid=44583561&page=1"));

        movie.id = String.valueOf(ID);
        movie.title = TITLE;
        movie.tag = TAG;
        movie.desc = DESC;
        movie.image = IMAGE;

        movieDetail.id = String.valueOf(ID + 3);
        movieDetail.movieId = String.valueOf(ID);
        movieDetail.title = TITLE;
        movieDetail.tag = TAG;
        movieDetail.desc = DESC;
        movieDetail.image = IMAGE;

        movieDetail.introduces = new ArrayList<>();
        movieDetail.introduces.addAll(introduceList);

        movieDetail.videos = new ArrayList<>();
        movieDetail.videos.addAll(videoList);
    }
}

File: createJson/src/main/java/com/ruzhan/jsonfile/data/WeGotNext.kt
package com.ruzhan.jsonfile.data

import com.ruzhan.jsonfile.model.Introduce
import com.ruzhan.jsonfile.model.Movie
import com.ruzhan.jsonfile.model.MovieDetail
import com.ruzhan.jsonfile.model.Video
import java.util.*

object WeGotNext {

    private const val ID = 1303
    private const val PLAY_COUNT = "1"

    private const val TITLE = "We Got Next 户外篮球纪录片"
    private const val TAG = "篮球训练·纪录片"

    private const val IMAGE = "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/we-got-next.jpg"

    private const val DESC = "《体育画报》五人组奔赴美国20天，只为探索一个美国街头篮球的真相。"

    val movie = Movie()
    val movieDetail = MovieDetail()

    private val introduceList = ArrayList<Introduce>()
    private val videoList = ArrayList<Video>()

    init {
        introduceList.add(Introduce(Introduce.TEXT,
                DESC, ""))

        introduceList.add(Introduce(Introduce.IMAGE, "",
                "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/we-got-next01.jpg"))

        introduceList.add(Introduce(Introduce.TEXT,
                "接下来的36分钟，让我们一起踏上美国最传奇的球场，与传奇对话。愿36分钟后，当你重新投出手中的篮球，感觉从此不同。", ""))

        videoList.add(Video((ID + 11).toString(),
                "We Got Next 户外篮球纪录片", IMAGE, PLAY_COUNT, 1, "电影",
                "http://player.bilibili.com/player.html?aid=26016313&cid=44583561&page=1"))

        movie.id = ID.toString()
        movie.title = TITLE
        movie.tag = TAG
        movie.desc = DESC
        movie.image = IMAGE

        movieDetail.id = (ID + 3).toString()
        movieDetail.movieId = ID.toString()
        movieDetail.title = TITLE
        movieDetail.tag = TAG
        movieDetail.desc = DESC
        movieDetail.image = IMAGE

        movieDetail.introduces = ArrayList()
        movieDetail.introduces.addAll(introduceList)

        movieDetail.videos = ArrayList()
        movieDetail.videos.addAll(videoList)
    }
}
